src/core/qgismobileapp.cpp

```diff
@@ -123,6 +125,21 @@ QgisMobileapp::QgisMobileapp( QgsApplication *app, QObject *parent )
 
   initDeclarative();
 
+  QSettings settings;
+#if defined(Q_OS_ANDROID)
+  const bool firstRunFlag = settings.value( QStringLiteral( "/QField/FirstRunFlag" ), QString() ).toString().isEmpty();
+  if (firstRunFlag)
+  {
+    QList<QPair<QString,QString>> projects;
+    QString path = AndroidPlatformUtilities().packagePath();
```

Comment:
If we use this, can we drop the `#if`?

```suggestion
    QString path = mPlatformUtils.packagePath();
```